service/build.gradle

```diff
@@ -209,6 +212,11 @@ dependencies {
 
   compile group: 'javax.xml.bind', name: 'jaxb-api', version: '2.3.1'
 
+  //overriding com.fasterxml.jackson.core:jackson-databind to fix CVE issues suppressed
+  compile group: 'com.fasterxml.jackson.core', name: 'jackson-core', version: '2.10.0'
```

Comment:
I'd be carefully using both Jackson 1 and Jackson 2. They might not be compatible.

Above comment assumes that spring boot brings other jackson modules in version 1.x.